tfexec/init_test.go

```diff
@@ -2,12 +2,17 @@ package tfexec
 
 import (
 	"context"
+	"runtime"
 	"testing"
 
 	"github.com/hashicorp/terraform-exec/tfexec/internal/testutil"
 )
 
-func TestInitCmd(t *testing.T) {
+func TestInitCmd_v012(t *testing.T) {
```

Comment:
Nit: Should this also have a v1 to ensure coverage? Seems like it'd be nice if the tests each had a mini-matrix of Terraform versions and expected behaviors, although it'd be a lot of effort with (potentially) little gain.